content/01.abstract.md

```diff
@@ -2,6 +2,13 @@
 
 Open, collaborative research is a powerful paradigm that can immensely strengthen the scientific process by integrating broad and diverse expertise.
 However, traditional research and multi-author writing processes break down at scale.
-New tools and workflows that rely on automation can catch errors and ensure that contributions are fairly tracked in massively collaborative research.
-We present techniques for overcoming challenges of open research, with special emphasis on manuscript writing.
-These include approaches for managing distributed authors and our new software, named Manubot, for automating citation and many other aspects of manuscript building.
+We present a new software tool named Manubot, available at <https://manubot.org>, for open scholarly writing.
+Manubot adopts the contribution workflow used by many large-scale open source software projects to enable collaborative authoring of scholarly manuscripts.
+With Manubot, manuscripts are written in Markdown and stored in a Git repository to precisely track changes over time.
+By hosting manuscript repositories publicly, such as on GitHub, multiple authors can simultaneously propose and review changes.
+Publication with Manubot is continuous.
+When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page.
+Manubot automates bibliographic tasks by implementing citation by identifier, where users cite persistent identifiers, whose metadata is then retrieved and rendered to a specified style.
+Manubot modernizes publishing to align with the ideals of open science, making it transparent, reproducibile, immediate, versioned, collaborative, and free of charge.
+Yet, fully open manuscript preparation also presents new challenges, which we faced while orchestrating Manubot's first use case: an open review article with {{deep_review_authors}} authors.
```

Comment:
This is a long sentence but I don't see how to break it up.

Reply:
I think this is fine, but note that only half the authors of this paper were associated with Deep Review. This could be rephrased but eliminating "we" and just saying `an issue that was faced in Manubot's first use case...`

Reply:
I think that saying we doesn't necessarily mean everyone and provides the context that some of us are writing based on our experience with Deep Review rather than as outside parties. I also like the active voice. @agitter do you think this is issue?

Reply:
Done in https://github.com/greenelab/meta-review/pull/211/commits/b95a292b9e7b315cafa57e460717182a98d952f0.

I had put those two closing sentences since I thought there would be a strong desire to emphasize Deep Review and the challenges there. Since we started this manuscript more as a discussion as those challenges and it evolved to be more about Manubot generally. I don't have a strong preference, and prefer the stronger closing statement for the abstract with those 2 sentences gone.